manual/vscode-extensions/vscode-ember-frost/vscode-ember-frost.nuspec

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>vscode-ember-frost</id>
+    <title>Ember Colorizer and Theme Extension</title>
+    <version>1.0.0</version>
+    <authors>Ciena Blue Planet</authors>
+    <owners>gep13, chocolatey</owners>
```

Comment:
Small one: chocolatey should be listed first by [§1.2.4](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#124-add-chocolatey-among-owners)